scripts/dev.js

```diff
@@ -127,10 +127,10 @@ class ActionServer extends BaseScript {
       if (hasFrontend) {
```

Comment:
also if not hasBackend we don't want to build & deploy actions + don't need to start the openwhisk server